spec/requests/chambers/create_files_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+describe "post /chambers/documents" do
+  it "creates new image for candidate" do
+    laurel = create(:laurel_peer)
+    sign_in(laurel.user)
+    cand = create(:candidate)
+    expect(Document.count).to eq(0)
+    post '/chambers/documents', params: { :document => {candidate_id: cand.id, document: 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7', name: 'Name', description: 'Description'} }
```

Comment:
Are you using data URLs in "reality" or only in the specs? If only in the specs, put this GIF in a fixture and upload that instead. If you're using data URLs elsewhere, where?